common/persistence/dataInterfaces.go

```diff
@@ -127,6 +127,7 @@ type (
 		ExecutionContext             []byte
 		State                        int
 		CloseStatus                  int
+		LastFirstEventID             int64
```

Comment:
You need persistence changes to read/write this new field.

Reply:
done